sys-process/btop/btop-1.2.13-r2.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs xdg-utils
+
+DESCRIPTION="A monitor of resources"
+HOMEPAGE="https://github.com/aristocratos/btop"
+SRC_URI="https://github.com/aristocratos/btop/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~ppc64 ~riscv ~x86"
+
+PATCHES=(
+	# Backported fixes for https://bugs.gentoo.org/884005,
+	# can be removed in 1.2.14 or later
+	"${FILESDIR}/${P}-fix-makefile-deps.patch"
+	"${FILESDIR}/${P}-verbose-mkdir.patch"
+
+	# Backported fix for compilation with clang 16.
+	# can be removed in 1.2.14 or later
+	"${FILESDIR}/${P}-allow-clang.patch"
```

Comment:
Since this patch does a bit more than just add support for Clang 16,
this description is a bit unsuited.
Maybe expand the description to include the note about the backported musl fix and links to the related bugs?